src/index.rs

```diff
@@ -802,7 +803,7 @@ mod tests {
 
             assert_eq!(actual.collisions.len(), 0);
             assert_eq!(actual.size(), 2);
-
+            #[cfg(target_os = "unix")]
             file.set_permissions(Permissions::from_mode(0o222))
                 .expect("Should be fine");
```

Comment:
Curious, what is it about?

Reply:
It's using the PermissionExt https://doc.rust-lang.org/std/os/unix/fs/trait.PermissionsExt.html which is only available on unix platforms

Reply:
Ah, I haven't looked at name of the test.. All is clear now.